Unio/Dependency.swift

```diff
@@ -29,6 +29,7 @@ public final class Dependency<Input: InputType, State: StateType, Extra: ExtraTy
     }
 
     /// Makes possible to get Observable from `Input`.
+    @available(*, deprecated, message: "Use `inputObservables.xxx` on Swift5.1 or greater. If Swift4.2, use `inputObservables[dynamicMember: \\.xxx]`.")
```

Comment:
Swift 5.0 is not included.
```suggestion
    @available(*, deprecated, message: "Use `inputObservables.xxx` on Swift5.1 or greater. Otherwise, use `inputObservables[dynamicMember: \\.xxx]`.")
```

Or could we just allow <5.1 users to use this method?
```suggestion
    @available(swift, deprecated: 5.1, message: "...")
```

Reply:
Fixed with `@available(swift, deprecated: 5.1, message: "...")` at 1a36a0bc4bb81b92bc075a85873d7d397dd4788a